<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameDialog</class>
 <widget class="QDialog" name="NewGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>178</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="StartButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BackButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>296</width>
     <height>136</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Greenlabel">
        <property name="text">
         <string>Green  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="GreencomboBox"/>
      </item>
      <item>
       <widget class="QLineEdit" name="GreenlineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Bluelabel">
        <property name="text">
         <string>Blue     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BluecomboBox"/>
      </item>
      <item>
       <widget class="QLineEdit" name="BluelineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Redlabel">
        <property name="text">
         <string>Red     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="RedcomboBox"/>
      </item>
      <item>
       <widget class="QLineEdit" name="RedlineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="Yellowlabel">
        <property name="text">
         <string>Yellow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="YellowcomboBox"/>
      </item>
      <item>
       <widget class="QLineEdit" name="YellowlineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="testLoad">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BackButton</sender>
   <signal>clicked()</signal>
   <receiver>NewGameDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartButton</sender>
   <signal>clicked()</signal>
   <receiver>NewGameDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
